source/hassio/installation.markdown

```diff
@@ -31,12 +31,17 @@ The following will take you through the steps required to install Home Assistant
 
 2. Install Home Assistant:
 
-   - Flash the downloaded image to an SD card using [balenaEtcher][balenaEtcher]. If using a Pi, we recommend at least a 32 GB SD card to avoid running out of space.
-   - Load the appliance image into your virtual machine software. (Note: You are free to assign as much resources as you wish to the VM, please assign enough based on your add-on needs)
+   - **Raspberry Pi, Tinkerboard, or an ODROID**:
+   Flash the downloaded image to an SD card using [balenaEtcher][balenaEtcher]. If using a Pi, we recommend at least a 32 GB SD card to avoid running out of space.
+   - **Virtual Appliance**:
```

Comment:
Emphasis should not be used in cases like this. This usually indicates headings are needed.